Package.swift

```diff
@@ -14,10 +14,10 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-argument-parser", .upToNextMinor(from: "1.1.2")),
-        .package(url: "https://github.com/apple/swift-syntax", .upToNextMinor(from: "509.0.1")),
-        .package(url: "https://github.com/apple/swift-tools-support-core.git", from: "0.2.3"),
-        .package(url: "https://github.com/jpsim/Yams.git", from: "5.0.0")
+        .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMinor(from: "1.5.0")),
+        .package(url: "https://github.com/apple/swift-syntax.git", .upToNextMinor(from: "510.0.3")),
```

Comment:
Not sure if related but the swift related repos have been migrated to under `swiftlang/` from `apple/` so please update them according to it.
Also, can you use `exact` version instead of `upToNextMinor` or `from`?

Reply:
Good catch. Not related it turned out, updated: 3dbb13769adc586a5edd0e1c01c6b69fdacbe8a1